src/reducers/poisReducer.js

```diff
@@ -37,16 +38,26 @@ const poisReducer = (state = initialState, action) => {
         ...state,
         specificPoiInfos: action.specificPoiInfos,
       };
+    case 'DISPLAY_FILTER_PAGE':
+      return {
+        ...state,
+        filterKeywordPageDisplay: action.filterKeywordPageDisplay,
+      };
     case 'HANDLE_SEARCHBAR_INPUT':
       return {
         ...state,
         searchBarValueInput: action.searchBarValueInput,
       };
-    case 'HANDLE_SUBMIT_SEARCHBAR':
+    case 'HANDLE_KEYWORD_FILTERING':
       return {
         ...state,
         filteredPoiByKeyword: action.filteredPoiByKeyword,
       };
+    case 'CLOSE_FILTER_COMPONENT':
```

Comment:
if you close it, it is always change in false ? If yes, you could change it as false directly in the reducer than in the dispatch

Reply:
Gotcha, changed it to a permanent falsen and suppressed the corresponding actions in the component